docs/infrastructure/requesting-infrastructure.md

```diff
@@ -51,3 +51,19 @@ Please be aware that:
 - the Infrastructure Team will initially setup the resource in a secure manner
   and provide guidance on how to keep it that way, however it is your
   responsibility to ensure that the resource remains secure
+
+## On Call availability
+
+It is expected that during some events within the competition calendar there
+will be a need for members of the Infrastructure Team to be available to quickly
+respond to any infrastructure issues.
```

Comment:
Should we note that the event organiser will need to make sure the on call volunteers are not doing tasks that cannot be stopped should they attend in person

Reply:
Is that needed here? Feels like that's going to be a natural part of agreeing for it to happen.
(not sure that that's necessarily specific to their location -- if they're a remote livestream producer and also the on-call for the website that could equally present challenges)

Reply:
Does e101b8f help?
Very open to better wordings for this - I struggled to make this succinct without it becoming a mess of if-this-and-also-this words.

Reply:
That wording looks good to me